pom.xml

```diff
@@ -71,7 +71,7 @@
         <netty.version>4.1.6.Final</netty.version>
         <slf4j.version>1.7.12</slf4j.version>
         <!-- If compiling with different hadoop version also modify default hadoop coordinates in TaskConfig.java -->
-        <hadoop.compile.version>2.3.0</hadoop.compile.version>
```

Comment:
@dclim i havn't run into this issue, maybe because i am not loading the s3a module, in fact i am using hdfs module to read segments.